Connected/Assets/Scripts/WirePhysics.cs

```diff
@@ -113,31 +112,55 @@ private class Stick {
 
 		UpdateStickLength();
 
-		foreach (Point p in points) {
+		var shuffledIndices = Enumerable.Range(0, points.Length).ToList().OrderBy(e => Random.value);
+		foreach (int index in shuffledIndices) {
+			Point p = points[index];
 			if (!p.locked) {
 				Vector3 positionBeforeUpdate = p.position;
-				p.position += (p.position - p.prevPosition) * 0.98f;
+				p.position += p.position - p.prevPosition;
 				p.position += Physics.gravity * Time.fixedDeltaTime * Time.fixedDeltaTime;
 				p.prevPosition = positionBeforeUpdate;
 			}
 		}
 
 		for (int i = 0; i < numIterations; ++i) {
-			foreach (Stick stick in sticks) {
+			shuffledIndices = Enumerable.Range(0, sticks.Length).ToList().OrderBy(e => Random.value);
+			foreach (int index in shuffledIndices) {
+				Stick stick = sticks[index];
 				Vector3 stickCenter = (stick.pointA.position + stick.pointB.position) / 2;
 				Vector3 stickDirection = (stick.pointA.position - stick.pointB.position).normalized;
 				
 				if (!stick.pointA.locked) {
 					stick.pointA.position = stickCenter + stickDirection * stickLength / 2;
-					stick.pointA.position = new Vector3(stick.pointA.position.x, Mathf.Clamp(stick.pointA.position.y, minimumY + wireRenderer.GetRadius(), float.PositiveInfinity), stick.pointA.position.z);
+					stick.pointA.position = new Vector3(stick.pointA.position.x, stick.pointA.position.y, stick.pointA.position.z);
 				}
 
 				if (!stick.pointB.locked) {
 					stick.pointB.position = stickCenter - stickDirection * stickLength / 2;
-					stick.pointB.position = new Vector3(stick.pointB.position.x, Mathf.Clamp(stick.pointB.position.y, minimumY + wireRenderer.GetRadius(), float.PositiveInfinity), stick.pointB.position.z);
+					stick.pointB.position = new Vector3(stick.pointB.position.x, stick.pointB.position.y, stick.pointB.position.z);
+				}
+			}
+
+			// Look for collisions from gravity force.
+			shuffledIndices = Enumerable.Range(0, points.Length).ToList().OrderBy(e => Random.value);
+			foreach (int index in shuffledIndices) {
+				Point p = points[index];
+				if (!p.locked) {
+					Vector3 positionBeforeUpdate = p.position;
+
```

Comment:
This variable seems to be unused. Remove if you want, otherwise it's not really a problem

I hate github's comment system.  When adding a single comment, the selected line seems to not be included and a few lines below will be, but when reviewing the selected line will be included and a few lines above..?